test/marketing_event_test.rb

```diff
@@ -46,4 +46,23 @@ def test_count_marketing_events
     marketing_events_count = ShopifyAPI::MarketingEvent.get(:count)
     assert_equal 2, marketing_events_count
   end
+
+  def test_add_engagements
```

Comment:
you could refactor this to 

` def test_add_engagements
     fake "marketing_events/1", method: :get, body: load_fixture('marketing_event')
     marketing_event = ShopifyAPI::MarketingEvent.find(1)
     engagement_fixture = load_fixture('engagement')
     engagements = ActiveSupport::JSON.decode(engagement_fixture)['engagements']
     fake "marketing_events/1/engagements", method: :post, status: 201, body: engagement_fixture
    
     assert marketing_event.add_engagements(engagements[0])
   end`